src/components/members/detail/GroupSection/index.tsx

```
@@ -46,16 +55,45 @@ const GroupSection = ({ profile, meId, memberId }: GroupSectionProps) => {
+                const endIndex = Math.min(startIndex + ITEMS_PER_ROW, meetingList.length);
+                const itemsInRow = meetingList.slice(startIndex, endIndex);
+
+                return (
+                  <MemberMeetingCardWrapper
+                    data-index={virtualRow.index}
+                    style={{
+                      transform: `translateY(${virtualRow.start}px)`,
+                      height: `${virtualRow.size}px`,
+                    }}
+                  >
+                    {itemsInRow.map((meeting) => (
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)